workflow-ui/core-common/src/main/java/com/squareup/workflow1/ui/ViewRendering.kt

```diff
@@ -0,0 +1,4 @@
+package com.squareup.workflow1.ui
+
+@WorkflowUiExperimentalApi
+interface ViewRendering
```

Comment:
I still don't see the point of this class. Why shouldn't I be able to create an `AndroidViewContainer<String>` if I want to?